test/os/os.mock.js

```diff
@@ -127,6 +128,8 @@ beforeEach(function() {
     if (!os.settingsManager) {
       os.settingsManager = os.ui.config.SettingsManager.getInstance();
     }
+
+    os.time.replacers.init();
```

Comment:
This feels like an odd place to also include and call this. Is there some reason we need it here and in `MainCtrl`?

Reply:
Tests don't load `MainCtrl`, so this file sets up managers and such that would normally be done there.

Reply:
That might be more appropriate in `os.mock.js` then if its for tests.